.yamato/_abv.yml

```diff
@@ -0,0 +1,356 @@
+    artifacts:
+        logs:
+            paths:
+              -  "**/test-results/**"
+smoke_test_editmode_2019.3:
+    name: SRP Smoke Test - editmode_2019.3
+    agent:
+        type: Unity::VM
+        image: sdet/gamecode_win10:stable
+        flavor: b1.large
+    variables:
+        UPM_REGISTRY: https://artifactory-slo.bf.unity3d.com/artifactory/api/npm/upm-candidates
+    commands:
+      -  git clone git@github.cds.internal.unity3d.com:unity/utr.git TestProjects/SRP_SmokeTest/utr
+      -  pip install unity-downloader-cli --extra-index-url https://artifactory.internal.unity3d.com/api/pypi/common-python/simple --upgrade
+      -  cd TestProjects/SRP_SmokeTest && unity-downloader-cli --source-file ../../unity_revision.txt -c editor --wait --published-only
+      -  cd TestProjects/SRP_SmokeTest && utr\utr --suite=editor --platform=editmode --testproject=. --editor-location=.Editor --artifacts_path=test-results
+    dependencies:
+      -  path: .yamato/_editor.yml#editor:priming:2019.3:windows
+         rerun: on-new-revision
+    artifacts:
+        logs:
+            paths:
+              -  "**/test-results/**"
+all_smoke_tests_2019.3:
+    name: All Smoke Tests - 2019.3
+    dependencies:
+      -  path: .yamato/_abv.yml#smoke_test_Standalone_2019.3
+         rerun: on-new-revision
+      -  path: .yamato/_abv.yml#smoke_test_playmode_2019.3
+         rerun: on-new-revision
+      -  path: .yamato/_abv.yml#smoke_test_editmode_2019.3
+         rerun: on-new-revision
+all_project_ci_2019.3:
+    name: _ABV for SRP repository - 2019.3
+    dependencies:
+      -  path: .yamato/_packages.yml#all_package_ci_2019.3
+         rerun: on-new-revision
+      -  path: .yamato/all-universal.yml#All_Universal_2019.3
+         rerun: on-new-revision
+      -  path: .yamato/all-universal_stereo.yml#All_Universal_Stereo_2019.3
+         rerun: on-new-revision
+      -  path: .yamato/all-shadergraph.yml#All_ShaderGraph_2019.3
+         rerun: on-new-revision
+      -  path: .yamato/all-hdrp.yml#All_HDRP_2019.3
+         rerun: on-new-revision
+      -  path: .yamato/all-vfxmain.yml#All_VFXMain_2019.3
+         rerun: on-new-revision
+      -  path: .yamato/all-vfx_lwrp.yml#All_VFX_LWRP_2019.3
+         rerun: on-new-revision
+all_project_ci_nightly_2019.3:
+    name: _Nightly ABV against 2019.3
+    dependencies:
+      -  path: .yamato/_abv.yml#all_project_ci_2019.3
+         rerun: on-new-revision
+      -  path: .yamato/_abv.yml#smoke_test_Standalone_2019.3
+         rerun: on-new-revision
+      -  path: .yamato/_abv.yml#smoke_test_playmode_2019.3
+         rerun: on-new-revision
+      -  path: .yamato/_abv.yml#smoke_test_editmode_2019.3
+         rerun: on-new-revision
+      -  path: .yamato/universal-android-opengles3.yml#Universal_Android_OpenGLES3_Standalone_2019.3
+         rerun: on-new-revision
+      -  path: .yamato/universal-android-vulkan.yml#Universal_Android_Vulkan_Standalone_2019.3
+         rerun: on-new-revision
```

Comment:
android targets arent running on 7.x.x/release. i think we should exclude them from this backport PR and add them back in at another time

Reply:
solved this with the last commit